### DISCUSSION TITLE
Github Copilot Student Access Issue

### DISCUSSION THREAD

**Discussion by Flvmbo** [community/community#50842] [GitHub Education] [answered]
Hey everyone, 

I am a verified student on Github Student Developer Pack and I am trying to access the Copilot because I believe it is free for students but for whatever reason I see the 2 month trial page when checking the Copilot tab in my account settings. I reverified my studentship yesterday and it was approved but I still don't have access to Copilot. I want to utilize it because I have heard it is very helpful.

Please help! 

EDIT: For whatever reason, after fiddling with it for a while it just started working

**Comment by Flvmbo** [comment, marked as answer]
For whatever reason, after fiddling with it for a while it just started working

**Reply by pppppkun** [reply]
Hi, I met this problem too. How long did you wait for it work normally?

**Reply by Flvmbo** [reply]
> Hi, I met this problem too. How long did you wait for it work normally?

About 24 hours

**Reply by WilCF** [reply]
I've got the same problem. I click 'Start Free Trial' and the page just sits there...

**Reply by ahmadi2002taha-web** [reply]
(I got a student account and it was approved, but it doesn't work for me and it's still free, why?)

**Comment by denizyennerr** [comment]
I am trying to access the Copilot, but for whatever reason, I cannot go through the location verification by Copilot, although I have allowed permission to GitHub to access my location. I reverified my studentship, and it was approved, but I still don't have access to Copilot. Can someone maybe guide or help me with this issue?
